### ISSUE TITLE
Symfony ContextErrorException : unserialize Error at offset (DataCollector) - Error 500 when debug enabled

### ISSUE THREAD

**Issue by ghost** [PrestaShop/PrestaShop#22231]
#### Describe the bug

Error 500 when debug enabled php 7.3.25, if i use php 7.2 no error

#### Steps to Reproduce

1. Make a new installation.
2. Enable Debug Mode
3. Use php 7.3.25
4. Go to the orders list (look at the error)
5. Go to the customers list (look at the error)
6. Go to the categorys list (look at the error)

7. **Now go to the product list there are no error**

**Configuration**

I use a server with Plesk on debian

![Capture d’écran de 2020-12-04 00-25-06](https://user-images.githubusercontent.com/5064590/101101144-3abb0b00-35c8-11eb-9076-9641e120184e.png)

**Error in console Tab**

![Capture d’écran de 2020-12-04 00-25-37](https://user-images.githubusercontent.com/5064590/101101185-50c8cb80-35c8-11eb-87ad-5879a0e0c5bd.png)

**Error in Network Tab**

A lot of memory 

![Capture d’écran de 2020-12-04 00-25-46](https://user-images.githubusercontent.com/5064590/101101273-8077d380-35c8-11eb-8a44-8c3e9aca015f.png)

#### Additional information

I found a similar issue https://github.com/PrestaShop/PrestaShop/issues/16873

I don't have this error on all new controller.

```
CRITICAL00:53:28 | php | Uncaught Notice: unserialize(): Error at offset 315894 of 1800983 bytes
-- | -- | --
CRITICAL00:53:28 | request | Uncaught PHP Exception Symfony\Component\Debug\Exception\ContextErrorException: "Notice: unserialize(): Error at offset 315894 of 1800983 bytes" at /var/www/vhosts/ee.wtf/httpdocs/1770/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/DataCollector/DataCollector.php line 56
```

* PrestaShop version: 1.7.7.0
* PHP version:  7.3.25

**Comment by matks** [comment]
Hi @okom3pom so you have multiple different issues here 😄 

- the JS error is because you are GDPR module V1.2.0 which is not compatible with PS 1.7.7.0 ; you need the v1.2.1
- the HTTP 500 issue I dont know, we need more infos
- the `Uncaught Notice: unserialize(): Error at offset 315894 of 1800983 bytes` is Symfony debug tools trying to serialize your data to present them into the Symfony Debug bar and failing because you have too much data 😄

**Comment by ghost** [comment]
Hi @matks for the js i open an other issue : https://github.com/PrestaShop/PrestaShop/issues/22232

The error 500 is only when i use php 7.3

**Comment by ghost** [comment]
I work on a new install

**Comment by florine2623** [comment]
Hi @okom3pom & @matks ,
I did manage to reproduce the same issue on PS 1.7.7.0 fresh install + PHP 7.3 + debug mode ON.

Debug mode OFF, no error 500 :
![Capture d’écran 2020-12-04 à 11 42 26](https://user-images.githubusercontent.com/16019289/101154300-d7f85c80-3625-11eb-86a7-6f40a2b7fb57.png)

Debug mode ON : 
![Capture d’écran 2020-12-04 à 11 42 33](https://user-images.githubusercontent.com/16019289/101154309-dc247a00-3625-11eb-874d-ec50fc921b0d.png)
![Capture d’écran 2020-12-04 à 11 42 43](https://user-images.githubusercontent.com/16019289/101154316-de86d400-3625-11eb-83a9-c60a9d862087.png)
![Capture d’écran 2020-12-04 à 11 43 57](https://user-images.githubusercontent.com/16019289/101154455-07a76480-3626-11eb-9996-8c48dc0eef3a.png)

I’ll add this to the debug backlog so that it’s fixed.
If you have already fixed it on your end or if you think you can do it, please do send us a pull request!
Thanks.

**Comment by ghost** [comment]
Hi @florine2623 i think it's a migration page problem not only in the order page, the label migration is better?

And i don't understand why I haven't seen this when i tested the beta and RC

I try on a RC  when i have the time

**Comment by florine2623** [comment]
@okom3pom I will add migration label also, thanks!

I have tested this on beta1, beta2 and RC1, I have the same issue on the three fresh installs : PHP 7.3 + debug ON = error 500

Please see the screenshots below : 
**beta1 :**
![Capture d’écran 2020-12-04 à 15 14 56](https://user-images.githubusercontent.com/16019289/101174046-a2fb0280-3643-11eb-82e5-f1c9822a5619.png)

**beta2 :**
![Capture d’écran 2020-12-04 à 15 15 01](https://user-images.githubusercontent.com/16019289/101174076-a8f0e380-3643-11eb-8ab9-8b815a5a5b29.png)

**RC1 :**
![Capture d’écran 2020-12-04 à 15 15 06](https://user-images.githubusercontent.com/16019289/101174094-af7f5b00-3643-11eb-82bb-0a7988a2b1f6.png)

The issue is present on the 3 previous versions.
Also, when clicking on the web profiler, here's the error : 
![Capture d’écran 2020-12-04 à 15 15 29](https://user-images.githubusercontent.com/16019289/101174247-e9e8f800-3643-11eb-8ee2-e4f1aa2013cb.png)

**Comment by marionf** [comment]
@hibatallahAouadni Could you check if it's a regression ?

**Comment by khouloudbelguith** [comment]
Hi @marionf, @hibatallahAouadni,

The PS1.7.6.9 is not compatible with PHP7.3.

Thanks!

**Comment by matks** [comment]
Hi, from what I see in the screenshots there is 2 issues:

1. the JS error is this one https://github.com/PrestaShop/PrestaShop/issues/18776 it is fixed when using GDPR module v1.2.1

2. the HTTP 500 error you can see being displayed in the Console is Symfony Debug Bar that crashes because it tries to serialize an object too big and it crashes (not enough memory). This can happen with big orders data, but it's not an issue.

It's not an issue because it's just Symfony debug tool trying to compute some data to help the developer but the data is too big 😄  . Symfony is just too hungry and tries to process a data too big for it, nothing bad about that. The only consequence is that the request fail and the SF debug bar does not appear in debug mode.

**Comment by ghost** [comment]
Hi,

@matks i know 1.2.1 fix the JS error but the issue is : 

https://github.com/PrestaShop/PrestaShop/issues/22232

If prestashop don't propose update i don't think merchant goes download zip from github and install the zip manualy .

This issue concern only the 500 in the Console.

**Comment by matks** [comment]
I update the title to make it more accurate.

**Comment by hibatallahAouadni** [comment]
Hello @marionf 

As @khouloudbelguith mentioned above, it couldn't be a regression cause 1.7.6.9 is not compatible with 7.3.
So, I change the status to **`To Do`**, feel free to change it :sweat_smile: 

Thanks!

**Comment by ghost** [comment]
I complete the Issue with : 

Make a new installation.

Go to the orders list (look at the error)

Go to the customers list (look at the error)

Go to the categorys list (look at the error)

Now go to the product list there are no error

@matks say : 

```It's not an issue because it's just Symfony debug tool trying to compute some data to help the developer but the data is too big```

But a lot of listing give an error and not  a lot of datas

**Comment by Progi1984** [comment]
Fixed by #22688
